### ISSUE TITLE
[aur-auto-vote] error when trying to login

### ISSUE THREAD

**Issue by latipun7** [cryzed/bin#13]
I installed the script thru [AUR](https://aur.archlinux.org/packages/aur-auto-vote-git), but error when I run the command:
```log
❯ aur-auto-vote latipun
Password: 
Traceback (most recent call last):
  File "/usr/bin/aur-auto-vote", line 132, in <module>
    main(arguments)
  File "/usr/bin/aur-auto-vote", line 84, in main
    if not login(session, arguments.username, password):
  File "/usr/bin/aur-auto-vote", line 38, in login
    return bool(soup.select_one('#archdev-navbar').find('a', href=lambda h: h and h.rstrip('/').endswith('/logout')))
AttributeError: 'NoneType' object has no attribute 'find'
```
The response for session login is `<Response [422]>`. I think I type (paste) the password correctly. I can login in browser with same pasted password.

I don't know what happens. Thank you.

**Comment by MacroController** [comment]
Same. I have version `r146.bb877b8-1`, and have the exact same error as latipun7. My shell is `zsh`, if that matters.

**Comment by parcelcat** [comment]
Same here. I can reproduce the error on both `bash` and `zsh`.

**Comment by tronfy** [comment]
Seems to be fixed with #14, feedback is welcome.

**Comment by cryzed** [comment]
Thanks @tronfy for providing a fix for this issue!
